macros/utils/table_operations/insert_rows.sql

```diff
@@ -52,8 +52,13 @@
     {% for row in rows %}
       {% set rendered_column_values = [] %}
       {% for column in columns %}
-        {% set column_value = elementary.insensitive_get_dict_value(row, column.name) %}
-        {% do rendered_column_values.append(elementary.render_value(column_value)) %}
+        {% if column.name == "created_at" %}
```

Comment:
Maybe there should be a const, or given that it's dbt, a function (macro) that returns this const.

Reply:
Also could be that we need to provide this as value in the actual upload of the data rather than magically here.
The challenge is that we'd need to be able to tell it to be unquoted.